src/types/composer.ts

```diff
@@ -1,15 +1,36 @@
 import algosdk from 'algosdk'
-import { sendAtomicTransactionComposer } from '../transaction/transaction'
+import { encodeLease, encodeTransactionNote, sendAtomicTransactionComposer } from '../transaction/transaction'
 import { TransactionSignerAccount } from './account'
 import { AlgoAmount } from './amount'
+import { SendAtomicTransactionComposerResults } from './transaction'
+import Transaction = algosdk.Transaction
+import TransactionWithSigner = algosdk.TransactionWithSigner
+import isTransactionWithSigner = algosdk.isTransactionWithSigner
+import encodeAddress = algosdk.encodeAddress
 
-export type CommonTxnParams = {
+/** Transaction parameter that provides an account address. */
+export type AccountAddressParam = string
+
+/** Transaction parameter that provides a round number. */
+export type RoundNumberParam = bigint
+
+/** Transaction parameter that provides an asset amount. */
+export type AssetAmountParam = bigint
+
+/** Transaction parameter that provides an asset ID. */
+export type AssetIdParam = bigint
+
+/** Transaction parameter that provides an application ID. */
+export type AppIdParam = bigint
```

Comment:
Adding explicit types for the kind of information being asked for - makes intellisense nicer / more obvious and allows us to see exactly what types we are accepting for each bit of information.

Reply:
I feel the other way around. `string` makes for nicer Intellisense than `AccountAddressParam`. I expect to see the type here, not a description of the value. One can read the TSDoc description to understand what the value should be.

Reply:
Removed for simplicity in [560195d](https://github.com/algorandfoundation/algokit-utils-ts/pull/261/commits/560195d8b12e03a6045444e24f891b61609b5e1a)